metrics/aocih.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+# The MIT License (MIT)
+#
+# Copyright (c) 2021-2024 Yegor Bugayenko
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+set -e
+set -o pipefail
+
+java_file=$1
+output=$2
+
+creation_timestamp=$(git log --diff-filter=A --format=%at -- "$java_file" | tail -1)
+if [[ -z "$creation_timestamp" ]]; then
+    age_in_hours=0
+else
+    current_timestamp=$(date +%s)
```

Comment:
@lueFlake "age of class in hours" is the number of hours since the date of birth of the repository. It has nothing to do with the current time.

Reply:
@yegor256 you mean difference between date of birth and date of class creation? i think i can redo it then

Reply:
@lueFlake yes, like this:

```
A = C - D
```

Here, A is the age of file, C is the time when the file was first time added to the repo, and D is the time when the first file was added to the repo.